src/cx/google/drive/drive.ts

```diff
@@ -0,0 +1,14 @@
+/// <reference path="../../../../typings/tsd.d.ts" />
+
+import {Files} from 'cx/google/drive/files';
+import {DriveQuery} from 'cx/google/drive/query';
+
+export {Files} from 'cx/google/drive/files';
+export {DriveQuery} from 'cx/google/drive/query';
+
+window['cx'] = window['cx'] || {};
+window['cx']['google'] = window['cx']['google'] || {};
```

Comment:
The typescript compiler will complain about unknown properties on already known objects.

The "workaround" for this is to declare the cx on the window object in a definition file, but I figured since this is used to expose the `cx` to < ES6 javascript it's better then adding it to the definition. If it's in the definition I figured that it might be telling anyone who uses this to use `window.cx.google.drive.files` instead of `import {Files} from 'cx/google/drive/files';` 

Makes sense?